org.lflang.tests/build.gradle

```
@@ -21,14 +21,14 @@ apply plugin: 'jacoco'
 
 jacoco {
     toolVersion = jacocoVersion
-    reportsDir = file("$buildDir/reports/jacoco") //default directory where jacoco generates test reports
+    reportsDirectory = file("$buildDir/reports/jacoco") //default directory where jacoco generates test reports
 }
 
 
 jacocoTestReport {
     reports {
-        xml.enabled true
-        csv.enabled true
+        xml.required = true
+        csv.required = true
```

We report to CodeCov using XML, so no need to produce CSV.
```suggestion
        csv.required = false
```